charts/linkerd2/Chart.yaml

```diff
@@ -9,3 +9,10 @@ name: "linkerd2"
 sources:
 - https://github.com/linkerd/linkerd2/
 version: 0.1.0
+maintainers:
```

Comment:
I *think* we'll want our github email addresses here. But, I'm not sure how this works in the tooling =)